docs/detections/alerts-view-details.asciidoc

```diff
@@ -83,12 +101,12 @@ When gathering threat intelligence data for an alert, {elastic-sec} queries the
 
 If these fields aren't available, {elastic-sec} does not perform the query and does not display threat intelligence data for the alert.
 
-Discovered threat indicator match data is placed under the **Threat Match Detected** subsection of the Overview tab.
+Discovered threat indicator match data is placed under the **Threat Match Detected** subsection of the Enriched data section.
```

Comment:
```suggestion
Discovered threat indicator match data appears under the **Threat Match Detected** subsection of the Enriched data section.
```
I'm not totally clear on what "Discovered" means here. Would "Relevant" or "Available" maybe be a better fit?

Reply:
```suggestion
Available threat indicator match data is placed under the **Threat Match Detected** subsection of the Enriched data section.
```